Add pdb-explorer extension

mattgra opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Could you move metadata images to a folder called `metadata` and reference them in  `README` instead

- Remove unused extension-icon.png

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

mattgra commented:
Hi @pernielsentikaer - thanks for the review!

As requested - changes are addressed in latest push
- [x] Created metadata folder and moved screenshots from `assets` to `metadata`
- [x] Removed unused extension-icon.png from `assets` folder
- [x] Referenced screenshots via links in `README` instead of showing screenshots directly

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
